src/components/Capture/capture.js

```diff
@@ -97,11 +98,15 @@ class Capture extends Component {
       return
     }
     payload = {...payload, documentType}
-    if (this.props.side === 'back' && !this.props.useWebcam) {
-      payload = {...payload, valid: true}
+    if (this.props.useWebcam) {
+      postToServer(this.createJSONPayload(payload), serverUrl, token,
+        (response) => this.onServerResponse(response),
+        (response) => this.onServerError(response)
+      )
+      this.setState({allowApiAdvancedValidation: false})
     }
     else {
-      postToServer(this.createJSONPayload(payload), serverUrl, token, (response) => this.onServerResponse(response), (response) => this.onServerError(response))
+      payload = {...payload, valid: true}
```

Comment:
This feels redundant. Why do we need the concept of a capture being valid anymore given we track whether or not to set advanced_validation?